pkg/cloudprovider/providers/gce/gce_alpha.go

```diff
@@ -23,7 +23,13 @@ import (
 )
 
 // All known alpha features
-var knownAlphaFeatures = map[string]bool{}
+var knownAlphaFeatures = map[string]bool{
+	GCEDiskAlphaFeatureGate: true,
+}
+
+const (
+	GCEDiskAlphaFeatureGate = "GCEDiskAlphaAPI"
```

Comment:
You want to gate it per feature, right? Or per API?

Reply:
When this alpha disk feature is used, we want the entire disk API to use alpha. Since that makes it not specific to a particular feature, I made the feature name here, generic enough that we can potentially reuse the same flag for future features.